src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -126,7 +129,11 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
       })
       // Assume fast fee for send max
       const fastFee = adapterFees[FeeDataKey.Fast]
-      const chainAsset = await getAssetData(asset.network)
+      const chainAsset = await getAssetData({
+        chain: ChainTypes.ETH,
+        network: NetworkTypes.MAINNET,
+        tokenId: address
```

Comment:
this `address` represents the send to asset. I think we would want to pull it off of asset.tokenId or asset.contractAddress (I don't think this exists)